src/current/_data/cloud_releases.csv

```diff
@@ -56,3 +56,4 @@ date,sha
 2023-07-24,null
 2023-07-26,null
 2023-07-31,null
+2023-08-01,null
```

Comment:
Should this date match the one in _config_cockroachdb.yml?

Reply:
no, that date is just used for when we updated the CockroachDB version cloud is running, which I also tacked onto this PR